### ISSUE TITLE
Suggestion: Push Badge into flow-component in 24.5

### ISSUE THREAD

**Issue by knoobie** [TatuLund/badge#5]
Now that https://github.com/vaadin/flow/pull/19568/files badge is officially in 24.5 by default: It's time to move your addon into flow-components, right? ;)

**Comment by TatuLund** [comment]
Yes. Also in my add-on I can drop the part that imports Badge styles even if they are not in the theme,json.
